appserver/pom.xml

```diff
@@ -191,7 +191,7 @@
         <version_suffix />
         <major_version>7</major_version>
         <minor_version>0</minor_version>
-        <update_version>0</update_version>
+        <update_version>1</update_version>
```

Comment:
1. for the master it should be 2 already
2. I think those should be calculated, maybe with [build-helper:parse-version](https://www.mojohaus.org/build-helper-maven-plugin/parse-version-mojo.html) or something else - to never need to update this manually again

Reply:
> I think those should be calculated, maybe with [build-helper:parse-version](https://www.mojohaus.org/build-helper-maven-plugin/parse-version-mojo.html) or something else - to never need to update this manually again

yeah, that would be better solution.

Reply:
> for the master it should be 2 already

I am aware of the master status, but want to call this out.

Reply:
> I am aware of the master status, but want to call this out.

I see. So new issue would be better choice to report this with. (And it does not require ECA to sign.)

Reply:
yeah, that is true. But for me to proceed on the homebrew PR, I should propose a known fix one (that is why this PR got created)

Reply:
And for some projects (not saying this one :) ), the feedback on issues are not faster than PRs. So I always prefer creating the PRs (I actually already signed the ECA XD )

Reply:
not sure why ECA still failing :(
![image](https://user-images.githubusercontent.com/1580956/215802925-274f1967-9219-48cb-95de-6d06f8121b5d.png)

Reply:
I revalidated the ECA, but unfortunately it keeps failing.

Reply:
ECA status might take some time to propagate to service.

This however is wrong change. There will be no 7.0.1 release, as it's already in central, and this will be out of date immediately. This will need manual intervention again, and I will not remember that in few hours.

Reply:
@pzygielo agree with you, closing this PR for now (as we have shipped the homebrew PR). :) 

Thanks @arjantijms for the revalidation, I will see if I can contribute a better patch.

Reply:
Appreciate the comments in here.